src/test-komodo/main.cpp

```diff
@@ -10,7 +10,9 @@ int main(int argc, char **argv) {
     assert(init_and_check_sodium() != -1);
     ECC_Start();
     ECCVerifyHandle handle;  // Inits secp256k1 verify context
+    SetupNetworking();
     SelectParams(CBaseChainParams::REGTEST);
+    chainName = assetchain(); // KMD by default
```

Comment:
Do we really need to explicitly re-initialize the `chainName` here? When we run `komodo-test`, the default constructor `assetchain::assetchain()` was already executed before `main()` from `init.cpp` and seems it worked fine:
```
Breakpoint 1, assetchain::assetchain (this=0x555556f93060 <chainName>) at ./assetchain.h:21
21          assetchain() : symbol_("") {}
(gdb) bt
#0  assetchain::assetchain (this=0x555556f93060 <chainName>) at ./assetchain.h:21
#1  0x00005555559bf9a8 in __static_initialization_and_destruction_0 (__initialize_p=1, __priority=65535)
    at init.cpp:111
#2  0x00005555559bfabc in _GLOBAL__sub_I_pzcashParams () at init.cpp:2127
#3  0x00007ffff7829ebb in call_init (env=<optimized out>, argv=0x7fffffffdea8, argc=1)
    at ../csu/libc-start.c:145
#4  __libc_start_main_impl (main=0x5555556156ca <main(int, char**)>, argc=1, argv=0x7fffffffdea8,
    init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fffffffde98)
    at ../csu/libc-start.c:379
#5  0x0000555555615345 in _start ()
```

Reply:
no, actually this statement is not needed, it's added just for clarity